packages/realm-app-importer/package.json

```diff
@@ -2,11 +2,16 @@
   "name": "realm-app-importer",
   "version": "0.1.0",
   "description": "Wrapping the Stitch CLI to import an app from a template (w. secrets)",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "./dist/bundle.cjs.js",
+  "module": "./dist/bundle.es.js",
+  "types": "./dist/bundle.d.ts",
+  "react-native": "./dist/bundle.react-native.es.js",
   "bin": "realm-app-importer",
   "scripts": {
-    "build": "tsc -p tsconfig.build.json",
+    "generate-types": "tsc --project tsconfig.types.json --declaration --emitDeclarationOnly --declarationDir types/generated",
+    "build": "npm run generate-types && rollup --config && npm run build-cli",
+    "build-cli": "tsc -p tsconfig.build.json",
```

Comment:
My rollup experience was quite low when I set this up.  I was not able to build the cli correctly with it, so I reverted to the previous compilation method.